frontend/src/i18n/locales/zh.json

```
@@ -0,0 +1,370 @@
+    "shopOurStory": "选购我们的故事",
+    "meetOurTeam": "认识团队",
+    "ourMission": "我们的使命",
+    "missionDescription": "让美观、可持续且实用的设计触手可及。我们相信您的家应反映您的价值观——简约、真实、精心策划。",
+    "missionQuote": "设计不仅在于外观和感觉，更在于其运作方式。我们创造的空间，为真实生活而美丽运作。",
+    "missionAuthor": "- Sarah Chen，创始人",
+    "ourStory": "我们的故事",
+    "ourValues": "核心价值观",
+    "valuesDescription": "这些原则指导着我们所做的一切，从产品选择到客户服务。",
+    "ourTeam": "我们的团队",
+    "teamDescription": "Minimal背后的热情团队，致力于为您带来卓越的设计和服务。",
+    "testimonials": "客户评价",
```

Biome (2.4.14)

[error] The key testimonials was already declared.

(lint/suspicious/noDuplicateObjectKeys)